blog/_src/posts/2024-05-17-racket-v8-13.md

```diff
@@ -1,28 +1,31 @@
     Title: Racket v8.13
-    Date: 2024-05-17T11:41:54
-    Tags: DRAFT
+    Date: 2024-05-16T23:06:42
+    Tags: release-announcement, DRAFT
+
+*posted by Stephen De Gabrielle*
 
 ----------------------------------------------------------------------
 
+# Racket v8.13 is now available
+
+We are pleased to announce Racket v8.13 is now available from [https://download.racket-lang.org/](https://download.racket-lang.org).
+
+
+## As of this release:
+
 - The racket/treelist and racket/mutable-treelist libraries provide
   list-like containers that support many operations in effectively
   constant time, including appending and extracting sub-lists without
   mutating the given list. Treelists are implemented as RRB Vectors,
-  invented by Stucki, Riompf, Ureche, and Bagwell.
+  invented by Stucki, Riompf, Ureche, and Bagwell. (see [4.16 Treelists](https://docs.racket-lang.org/reference/treelist.html#(part._treelist))
+and [RRB vector: a practical general purpose immutable sequence, ICFP 2015](https://dl.acm.org/doi/abs/10.1145/2784731.2784739))
 
-(see https://docs.racket-lang.org/reference/treelist.html#(part._treelist)
- and https://dl.acm.org/doi/abs/10.1145/2784731.2784739 )
- 
 - The `hash-filter-keys` and `hash-filter-values` functions allow users
-  to filter hashes using a predicate on either keys or values.
-
-(see https://docs.racket-lang.org/reference/hashtables.html#%28def._%28%28lib._racket%2Fhash..rkt%29._hash-filter-keys%29%29 )
+  to filter hashes using a predicate on either keys or values. (see [4.15 Hash Tables](https://docs.racket-lang.org/reference/hashtables.html#%28def._%28%28lib._racket%2Fhash..rkt%29._hash-filter-keys%29%29))
 
 - The `vector-extend` and `vector*-extend` functions provide a way
   to pre-populate the prefix of a newly allocated vector using the elements
-  of an existing vector.
-
-(see https://docs.racket-lang.org/reference/vectors.html#%28def._%28%28lib._racket%2Fvector..rkt%29._vector-extend%29%29 )
+  of an existing vector. (see [4.15 Hash Tables](https://docs.racket-lang.org/reference/hashtables.html#%28def._%28%28lib._racket%2Fhash..rkt%29._hash-filter-keys%29%29))
```

Comment:
This change is not correct. Could you fix it?

Reply:
done

Reply:
You are right. 
I think I have fixed both links. (I also fixed discourse post)
Look ok now?